### PR TITLE
feat(connect): add support for StellarClaimClaimableBalanceOp.

### PR DESCRIPTION
**Pull request by overcat**
## Description
feat(connect): add support for StellarClaimClaimableBalanceOp.
chore: update stellar-sdk to v11.2.2

## Related Issue

N/A
## Screenshots:
N/A

### CODE REVIEW

File: .github/workflows/connect-test-params.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - run: yarn install
       # todo: ideally reuse libs from build step
       - run: yarn build:libs
-      - run: './docker/docker-connect-test.sh node -p "${{ inputs.test-pattern }}" -f 2-latest -i ${{ inputs.methods }}'
```
Comment (overcat) [inline .github/workflows/connect-test-params.yml]: We need to temporarily switch to the latest firmware for testing.

Comment (mroz22) [reply .github/workflows/connect-test-params.yml]: could this be possibly replaced by adding a legacy result here? https://github.com/trezor/trezor-suite/blob/develop/packages/connect/e2e/__fixtures__/stellarSignTransaction.ts#L143

**Comment by mroz22** [comment]
looks good, if CI passes, I am merging it. Thanks

**Comment by overcat** [comment]
It goes green, do you need me to rebase fixup commit?

**Comment by mroz22** [comment]
nope, github should make it

**Pull request merged**